wagtail/admin/views/generic/history.py

```diff
@@ -165,7 +165,7 @@ def get_cell_context_data(self, instance, parent_context):
         return context
 
 
-class HistoryView(PermissionCheckedMixin, BaseListingView):
+class HistoryView(PermissionCheckedMixin, BaseObjectMixin, BaseListingView):
```

Comment:
Inheriting from `BaseObjectMixin` so that we get the `pk_url_kwarg` config for free, and to follow the generic `UsageView`.